### PR TITLE
Content

### PR DESCRIPTION
**Pull request by Usagib**
Hey Guys, we just finished our project!
opening pull request for code review.
Happy Coding!

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under review by Ahmed Mahfoudh TSE

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+  linters:
+    stylelint:
```
Comment (stratospherique) [inline .stickler.yml]: - This not the linter configuration that you should adapt. Check Microverse's [repo](https://github.com/microverseinc/linters-config/tree/master/css) and follow the instruction.

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Hi Team :clap: :clap: . Great work :100: :100: . However, some minor changes that prevents your project's approval needs to be made. check below :arrow_down: 

#### Page structure

##### Footer

* [ ] It contains centered links (content is not important).
> Just use links instead of plain text

### That's it. When finished, kindly resubmit another code review request :pray: :pray: 

###### PS: check comments below :arrow_down:

**Comment by sebastianlujan** [comment]
Hello team!, We finished the required changes, waiting for code review.
Happy Hacking🦖

**Comment by samgaco** [comment]
Pull request under review

File: index.html [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+        <a href="#">Photoshop</a>
+        <a href="#">Plugins</a>
+        <a href="#">React</a>
+        <a href="#">Responsive</a>
+        <a href="#">Web Design</a>
+        <a href="#">Service</a>
+        <a href="#">Workers</a>
+        <a href="#">Sketch</a>
+        <a href="#">Typography</a>
+        <a href="#">UI</a>
```
Comment (samgaco) [inline index.html:527]: :+1:

File: .stickler.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+linters:
+  stylelint:
+    config: 'stylelint.config.js'
```
Comment (samgaco) [inline .stickler.yml:3]: :+1:

**Review by samgaco — APPROVED** [review]
#### All the changes have been implemented, hence you are approved :100: You may merge to the master branch and submit the project completion form now :1st_place_medal:  Regards!

-------------------
Recommended moving forward :arrow_down: 
* **[For your next code review requests]**  Use meaningful names for your Pull Requests

-------------------
* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

File: .stickler.yml (hunk 1 above)
Comment (Usagib) [reply .stickler.yml]: Yes, in fact we usa a css gradient tool , and i copied the resulting gradient code, but we did as you advised, and all good and ready to go TY!

**Pull request merged**